drivers/include/cc110x/cc110x-internal.h

```diff
@@ -0,0 +1,202 @@
+/*
```

Comment:
ah, forget about it - when looking at the internal structure of the driver the separation doesn't hurt...